build-aux/toolchain/build/dependencies.sh

```diff
@@ -0,0 +1,49 @@
+# Check if all dependencies are installed
+CROSSTOOLCHAIN_DEPENDENCIES_TOOL="make wget tar xz gcc g++ "
+CROSSTOOLCHAIN_DEPENDENCIES_LIBS=""
+
+# Add gcc dependencies
+CROSSTOOLCHAIN_DEPENDENCIES_TOOL+="bison flex texindex "
+CROSSTOOLCHAIN_DEPENDENCIES_LIBS+="libgmp libmpc libmpfr "
+
+# Add llvm (optional) dependencies
+if [[ "$COMPILER" == "llvm" ]]; then
```

Comment:
Not a big fan of the `[[` syntax, `[` works as well and has more functional guarantees since it can be either a builtin or a coreutils binary (`/bin/[`).

Reply:
done